docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {ListItemTagUsingInfoListItemExample} from './ListItemTagUsingInfoListItemExample';
+
+const codeSnippet = `
+  <InfoListItem
+    icon={{name: 'battery-charging-full'}}
+    iconColor="#528AAE"
+    style={{backgroundColor: Colors.white[100]}}
```

Comment:
please remove style={{backgroundColor: Colors.white[100]}} from snippet and format the snippet